okapi-core/src/main/java/org/folio/okapi/logging/FolioLoggingContext.java

```diff
@@ -0,0 +1,62 @@
+package org.folio.okapi.logging;
```

Comment:
The class should be moved from okapi-core to okapi-common that already is a dependency of RMB: https://github.com/folio-org/raml-module-builder/blob/v30.2.4/pom.xml#L162

Reply:
done